games-strategy/openxcom/additional-files/openxcom.rdef

```diff
@@ -16,54 +16,32 @@ resource app_version {
-        $"9EBC76C2C2BD4BC236BC76C2C2C2DDBFB9C4C2C302C2DDC302C2C2C302C2C2C2"
-        $"2DC236C309C0F9C230C19EC48AC0DDC57BC216C55AC13DC57BC216C78846C7E5"
-        $"C269C7FDC28EC7FDC28EC814C2ACC837C329C827C2E0C84AC377C7A7C41AC81A"
-        $"C3C7C730C46AC588C3CEC67BC450C588C3CEC34EC548C340C54BC2E7C589C2E7"
-        $"C589C297C5C0C26FC608C26FC5EAC26FC6080437AFFFFAA7BFFEFFF3EF7FFAEF"
-        $"BF1AC26FC608C26FC608C26FC5EAC2E7C589C297C5C0C2E7C589C340C54BC34E"
-        $"C548C588C3CEC588C3CEC67BC450C7A7C41AC730C46AC81AC3C7C837C329C84A"
-        $"C377C827C2E0C7FDC28EC814C2ACC7FDC28EC7E5C269C78846C57BC216C57BC2"
-        $"16C55AC13DC309C0F9C48AC0DDC230C19EC302C2C2C22DC236C302C2C2C2DDBF"
-        $"B9C4C2BC76C2DDBC76C2C2BC76C2C2BD4BC236BC6FC0F9BD48C19EBAEEC0DDBA"
-        $"04C216BA20C13DBA04C216B7F046B793C269B77EC28EB77EC28EB765C2ACB744"
-        $"C329B752C2E0B72FC377B7D1C41AB75EC3C7B843C46AB9F0C3CEB8F8C450B9F0"
-        $"C3CEBC38C54BBC38C54BBCA4C58EBD06C5EDBCE8C5C4BD06C5EDBD1BC613BD1B"
-        $"C613BD1BC637BC3FC6D0BCD1C676BC3FC6D0BC38B9F3C84DB9F3C84DB900C7C6"
-        $"B7D8C801B84DC7ADB76AC854B744C8EEB739C8A3B754C93AB785C98CB76AC96F"
-        $"B785C98CB797C9ABB7F3C9F7B7F3C9F7B866CA2EBA08CA01B917CA31BA26CADD"
-        $"BC72CB22BAF4CB3DBD4CCA78BC83C952BD52C9DEBC83C952C93DBFC3C756C306"
-        $"C93DC306C952C306C952C231C9DEC310CB22C234CA78C48FCB3DC57BCA01C55D"
-        $"CADDC66BCA31C78CC9F7C71BCA2EC78CC9F7C7ECC9ABC7FDC98CC7FDC98CC816"
-        $"C96FC83EC8EEC82CC93AC84EC8A3C7ABC801C81DC854C73AC7ADC58FC84DC687"
-        $"C7C6C58FC84DC344C6D0C27CC62EC26FC613C6080411FABFFEFF03B982B911B9"
-        $"7BB90AB985B8B1B985B8B1B995B803B9ECB6A0B9B8B753BA88B496BFC3B391BC"
-        $"7AB391C30CB391C59DB6A0C4FFB496C5CFB753C604B8B1C5F2B803C604B8B1C6"
-        $"0BB90AC607B911C60BB911C60BB911C622BA3BC5B2BC7DC604BB5FC3FCBFE2BF"
-        $"C3C19EC202C197BD84C197B9D8BC7DBB8BBFE2B985BB5FB97BB911B966BA3BB9"
-        $"7BB911B982B911B982B911BA00BAFBBDCDBB7FBB6EBBCBBD4DB99AB982B911BB"
-        $"DFB8CAB982B9110003C607B911C607B911C3ADB8CAC1BFBB7FC240B99AC41EBB"
-        $"CBC607B911C58CBAFBC607B9110003C607B911C607B911C58CBAFBC1BFBB7FC4"
-        $"1EBBCBC240B99AC607B911C3ADB8CAC607B9110003B982B911B982B911BBDFB8"
-        $"CABDCDBB7FBD4DB99ABB6EBBCBB982B911BA00BAFBB982B9110413FABFFFFF3F"
-        $"C607B911C60BB90AC604B8B1C604B8B1C5F2B803C59DB6A0C5CFB753C4FFB496"
-        $"BFC3B391C30CB391BC7AB391B9ECB6A0BA88B496B9B8B753B985B8B1B995B803"
-        $"B985B8B1B97BB90AB982B911B982B911BBDFB8CABDCDBB7FBD4DB99ABB6EBBCB"
-        $"B982B911BA00BAFBB982B911B97BB911B97BB911B966BA3BB9D8BC7DB985BB5F"
-        $"BB8BBFE2BFC3C19EBD84C197C202C197C5B2BC7DC3FCBFE2C604BB5FC60BB911"
-        $"C622BA3BC60BB911C607B911C607B911C58CBAFBC1BFBB7FC41EBBCBC240B99A"
-        $"C607B911C3ADB8CAC607B911050A000100000A0101011001178122040A000202"
-        $"03000A01020405000A010106100117812204"
+       $"6E636966020369C7B905000C0604CBCEA1B9B2CDBAB82ACEA1B9B2D505B310CD"
```

Comment:
Oops, in line 19 we have 7 spaces (instead of one tab.)

Are you comfortable with git? If not, I can update your PR with this tiny "fix" and the `REVISION` bump.

Hmm, I just noticed that your changes are in your master branch.
That's not recommended because it makes it harder for you to update your master branch with new commits from the project's master branch, should you wish to update your master branch before the merge.
But don't worry, you can continue with this PR, and we will also be able to merge it too, with no extra work.

(Once this PR is merged, you will need to update your local master branch and then `git push --force` your local master to your remote master (on GitHub) to re-sync your GitHub's master branch with the projects master. BTW, in case you need help for that, please do not hesitate to ask.)

For your next PRs, you should use [feature-branches](https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches).

Reply:
I'll fix it and submit from a feature branch. Part of doing this is getting some familiarity with GitHub, so it's worth me figuring out how to do it right.